packages/playground/src/tests/r3f-stress-test/SpaceStress.theatre-project-state.json

```diff
@@ -0,0 +1,1155 @@
+{
```

Comment:
I'm not sure if it matters, but the rocket's animation is a bit off (the parts of the rocket are not moving together). If we won't share this example with people from our community that much, then it won't be a problem, but I just thought I'll point this out.

https://user-images.githubusercontent.com/43729152/182893243-d8297aff-db97-4177-a4ac-73f6b7dc6b04.mp4

Reply:
Yep, this has always been an issue. I'd love to see this glb file was fixed for both examples.